cubic_traj_param param
---
cubic_traj_coeffs coeff
